Sources/xcproj/XcodeProj.swift

```diff
@@ -26,6 +26,11 @@ public class XcodeProj {
         let pbxProjData = try Data(contentsOf: pbxprojPaths.first!.url)
         let plistDecoder = PropertyListDecoder()
         pbxproj = try plistDecoder.decode(PBXProj.self, from: pbxProjData)
+        if let projectName = pbxprojPaths.first!.projectName() {
```

Comment:
It might be confusing for someone looking at this code to know why this is necessary. What do you think about extracting this into a more descriptive method:

```swift
static class func updateProjectName(path: Path, pbxproj: PBXProj) {
 // Your logic
}
```
And maybe adding a comment on the method explaining that the attribute might be missing and it's needed to generate certain comments.

Reply:
I moved it to `Sources/xcproj/PBXProj+Helpers.swift` as helper method.